kai/service/llm_interfacing/model_provider.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, config: KaiConfigModels):
                         "decoding_method": DecodingMethod.SAMPLE,
                         # NOTE: probably have to do some more clever stuff regarding
                         # config. max_new_tokens and such varies between models
-                        "max_new_tokens": 4096,
+                        "max_new_tokens": 2000,
```

Comment:
why the decrease?

Reply:
Was having some issues, I'll set it back